src/Schema.php

```diff
@@ -25,4 +25,28 @@ public static function deactivate() {
   public static function uninstall() {
   }
 
+  /**
+   * Cron event callback to ensure proper database indexes.
+   */
+  public static function cron_ensure_back_in_stock() {
+    global $wpdb;
+    $ids = $wpdb->get_col($wpdb->prepare("SELECT ID FROM {$wpdb->posts}
+      INNER JOIN {$wpdb->postmeta} pm ON {$wpdb->posts}.ID = pm.post_id
+      LEFT JOIN {$wpdb->postmeta} moeve ON {$wpdb->posts}.ID = moeve.post_id AND moeve.meta_key LIKE %s
+      WHERE pm.meta_key = %s
+        AND pm.meta_value <= CURRENT_DATE()
```

Comment:
The current date should be injected as a query variable from PHP (`date('Y-m-d')`), because the date in MySQL should not be trusted.
```suggestion
        AND pm.meta_value <= %s
```